table/column.go

```diff
@@ -47,6 +47,8 @@ type Column struct {
 	*model.ColumnInfo
 	// If this column is a generated column, the expression will be stored here.
 	GeneratedExpr ast.ExprNode
+	// GetGeneratedExpr gets the generated column exprNode. If we want to rewrite the node, please use this function to avoid data race.
```

Comment:
```suggestion
	// GetGeneratedExpr gets the generated column exprNode. If you want to rewrite the node, please use this function to avoid data race.
```